Android/app/src/main/java/fashiome/android/fragments/ProductsRecyclerViewFragment.java

```diff
@@ -96,6 +98,12 @@ public void onItemClicked(RecyclerView recyclerView, int position, View v) {
             public void done(List<Product> products, ParseException e) {
                 if (e == null) {
                     Log.d("DEBUG", "Retrieved " + products.size() + " products");
+                    for(Product p: products) {
```

Comment:
placeholder logging because we still need to fix some serialization of user object